Neos.Neos/Classes/Domain/Service/SiteService.php

```diff
@@ -176,7 +182,7 @@ public function createSite(
 
         $siteServiceInternals = $this->contentRepositoryRegistry->buildService(
             $site->getConfiguration()->contentRepositoryId,
-            new SiteServiceInternalsFactory()
+            new SiteServiceInternalsFactory($this->workspaceService)
```

Comment:
With this change, creating sites stops working for me. Could you check to make sure that this does not break site creation?

Reply:
@pKallert what does "stops working" mean? do you get an exception?

Reply:
ah, nevermind, I can reproduce it, nasty proxies
> Neos\Neos\Domain\Service\SiteServiceInternalsFactory_Original::__construct(): Argument #1 ($workspaceService) must be of type Neos\Neos\Domain\Service\WorkspaceService, Neos\Flow\ObjectManagement\DependencyInjection\DependencyProxy given

Reply:
![Bildschirmfoto 2024-10-11 um 15 30 28](https://github.com/user-attachments/assets/a0b02e73-1d40-4e3c-ab2b-3de980b90b82)

Reply:
can you check again?

Reply:
Works now, thank you 😄